main.py

```diff
@@ -536,7 +536,41 @@ def populate_memes():
                     tmp, FEEDBACK_DEL_TIMER, error=True, call_msg=message
                 )
 
-
+    elif message.content.startswith("!patreon"):
+        is_donator = False
```

Comment:
The word is actually `donor` 😋

Reply:
You want me to change the command form !patreon to !donor, or are you refering to the role donor that already exists? Because here I just check the new command. I check later if the user is a donor.

Reply:
oh now I got it. lol